torchquad/tests/simpson_test.py

```diff
@@ -32,7 +32,7 @@ def test_integrate():
     # 3D Tests
     N = 1030301  # N = 101 per dim
```

Comment:
I think in the other test you changed it so that it breaks if the N-finding method doesn't work correctly, right? Maybe do that here as well?

Reply:
Copy that